src/main/app/claims/offerClient.ts

```diff
@@ -37,4 +37,13 @@ export class OfferClient {
       }
     })
   }
+
+  static countersignOffer (externalId: string, user: User): Promise<Claim> {
+    return request.post(`${claimStoreApiUrl}/${externalId}/offers/defendant/countersign`, {
+      body: '',
```

Comment:
do you need to specify body here?

Reply:
No idea, but other methods did, so I did the same here.